### PR TITLE
Swedish translation of v1.0.0

### PR DESCRIPTION
**Pull request by magol**

### CODE REVIEW

**Comment by olivierlacan** [comment]
Thanks @magol!

**Comment by olivierlacan** [comment]
@magol Something in this new file is causing a runtime error: `RuntimeError: can't modify frozen String`

Can you run `bundle exec middleman build --verbose` locally and check it out? I can't find an obvious reason.

File: source/sv/1.0.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+    %li
+      Vid lansering behöver du bara flytta innehållet i sektionen 
+      <code>Unreleased</code> till en ny versionspost.
+
+.bad-practices
+  %h3#bad-practices
+    %a.anchor{ href: "#bad-practices", aria_hidden: "true" }
+    Kan ändringsloggar vara dåliga?
+
+  %p Ja, här är några exempel på då de är mindre användbara.
```
Comment (Art4) [inline source/sv/1.0.0/index.html.haml]: I think the text must be in a new line.

**Comment by magol** [comment]
I have not test environment to test this, but I hope #171 fix the issue

File: source/sv/1.0.0/index.html.haml (hunk 1 above)
Comment (magol) [inline source/sv/1.0.0/index.html.haml]: @Art4 But this is exactly how it look in the English source text.

Comment (Art4) [inline source/sv/1.0.0/index.html.haml]: Ok. It was a wild guess. 😊

**Pull request merged**